modules/fundamental/src/sbom/endpoints/test.rs

```diff
@@ -1309,6 +1311,24 @@ async fn query_sboms_by_array_values(ctx: &TrustifyContext) -> Result<(), anyhow
     query(2, "authors>ZZZ").await;
     query(2, "organization").await;
     query(1, "tool: syft").await;
+    query(1, "versions=2.13.8.Final-redhat-00004").await;
+    query(1, "versions~2.13.8").await;
```

Comment:
Since you have the logic in the translator that aliases `versions` to `version`, you can add a test for that, too. Hopefully, that'll make the coverage report a little happier...
```suggestion
    query(1, "versions~2.13.8").await;
    query(1, "version~2.13.8").await;
```

Reply:
@jcrossley3, done.